fvm/src/kernel/mod.rs

```diff
@@ -88,6 +92,11 @@ pub trait Kernel:
     fn machine(&self) -> &<Self::CallManager as CallManager>::Machine;
 }
 
+/// TODO
+pub trait Validator {
```

Comment:
`ValidateOps`? Or, well, I'm not sure what we're using this for.

Reply:
Just following the current system (sort of). Current use is different than the originally intended, where Validator was a trait only implemented on Kernels that meant to have it. `ValidateOps` is a bit closer to what it does now though.